git_models.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['app', 'lib']
 
-  spec.add_dependency 'rails', '>= 4.2'
-  spec.add_dependency 'hobo_fields', '~> 3.0'
-  spec.add_dependency 'git_lib', '~> 1.2'
+  spec.add_dependency 'rails',        '>= 4.2'
```

Comment:
I've been adding `, '< 7'` to these dependencies to make sure we don't accidentally start pulling in rails 7 when it starts under developments. I think it's a good idea to keep it capped that way.

Reply:
done